package.json

```diff
@@ -61,28 +61,15 @@
 		"prern-bundle": "patch-package --patch-dir gutenberg/packages/react-native-editor/metro-patch",
 		"rn-bundle": "react-native bundle",
 		"postrn-bundle": "patch-package --reverse --patch-dir gutenberg/packages/react-native-editor/metro-patch",
-		"i18n:gutenberg": "./bin/i18n-download.sh gutenberg",
-		"i18n:gutenberg:force": "cross-env REFRESH_I18N=1 ./bin/i18n-download.sh gutenberg",
-		"bundle": "npm run clean; npm run bundle:js && npm run genstrings",
-		"prebundle:js": "npm run i18n:gutenberg:force",
+		"i18n:update": "./bin/i18n-update.sh jetpack ./jetpack/projects/plugins/jetpack/extensions layout-grid ./block-experiments/blocks/layout-grid",
+		"i18n:update:debug": "./bin/i18n-update.sh --debug jetpack ./jetpack/projects/plugins/jetpack/extensions layout-grid ./block-experiments/blocks/layout-grid",
+		"bundle": "npm run clean; npm run bundle:js",
+		"prebundle:js": "npm run i18n:update",
```

Comment:
Now when generating a bundle, we'll always update the translations and produce the localization strings files.